cli_tools/common/utils/storage/scratch_bucket_creator.go

```diff
@@ -81,7 +81,7 @@ func (c *ScratchBucketCreator) getBucketAttrs(fileGcsPath string, project string
 func (c *ScratchBucketCreator) getBucketAttrsFromInputFile(fileGcsPath string, project string,
 	fallbackZone string) (*storage.BucketAttrs, error) {
 
-	fileBucket, _, err := SplitGCSPath(fileGcsPath)
+	fileBucket, err := GetBucketNameFromGCSPath(fileGcsPath)
```

Comment:
hmm this has been merged... I think "getBucketAttrsFromInputFile" rely on SplitGCSPath to check whether it's a valid file path (bucket name + object name). If you need to create bucket by a bucket name, it would be better to make a separate function.

Reply:
I've changed SplitGCSPath to work for both bucket only paths and for bucket+object. I've added GetBucketNameFromGCSPath for this specific use case where only bucket name is needed.

Do you have any concerns with this solution?

Reply:
Oh I didn't notice the new function behaves the same as the previous function. It's perfect,